.github/ISSUE_TEMPLATE/custom-pull-request-template.md

```diff
@@ -0,0 +1,30 @@
+# Description
```

Comment:
Let's make the header smaller, so instead of using # Description, maybe use ## Description, it will use smaller header instead :P